lib/sentry/config.ex

```diff
@@ -77,13 +77,35 @@ defmodule Sentry.Config do
     get_config(:enable_source_code_context, default: false, check_dsn: false)
   end
 
-  def root_source_code_path do
```

Comment:
For the deprecation strategy, I think it would be preferable to keep the old method with the same behavior, but mark it as `@deprecated`?

I'm guessing its usage is rare, but I'd rather not remove it until the next major release.

Reply:
@mitchellhenke do you mean keeping both the new and the old versions? See if the changes in https://github.com/getsentry/sentry-elixir/pull/437/commits/52d32cfb787d0a94c15228450f407917c57e1335 is what you had in mind.

Reply:
Yep, exactly!